gtda/diagrams/features.py

```diff
@@ -525,3 +525,133 @@ def transform(self, X, y=None):
                                         self.n_values, self.n_values).\
             transpose((1, 0, 2, 3))
         return Xt
+
+
+@adapt_fit_transform_docs
+class Silhouette(BaseEstimator, TransformerMixin):
+    """Silhouettes of persistence diagrams.
+
+    Based on ideas in [1]_. Given a persistence diagram consisting of
+    birth-death-dimension triples [b, d, q], subdiagrams corresponding to
+    distinct homology dimensions are considered separately.
+    The silhouette of each subdiagram is the weighted average
+    of the piecewise-linear functions (fibres) used  to derive a landscape.
+    A silhouette is represented as a vector, with entries computed over
+    a evenly sampled locations from appropriate ranges
+    of the `filtration parameter <https://giotto.ai/theory>`_.
+
+     Parameters
+    ----------
+    order: float, optional, default: ``1``
+        The order of the persistence for the weighted average of fibres
```

Comment:
Not sure this explanation is very clear. Also there is an orphan "d" at the end of it.